src/components/Board/Board.js

```diff
@@ -29,7 +29,11 @@ const Board = ({ history, match }) => {
       setError(false);
     }).catch((err) => {
       if (err.status === 401) {
-        history.push('/login');
+        const redirect = {
```

Comment:
Can you please extract that to a function and reuse it on the Board component as well? It seems that the if block is exactly the same.